CHANGELOG.md

```diff
@@ -14,7 +14,7 @@
 * `resource/access_application`: Add support for `auto_redirect_to_identity` ([#730](https://github.com/cloudflare/terraform-provider-cloudflare/issues/730)) 
 * `resource/access_application`: Add CORS support ([#725](https://github.com/cloudflare/terraform-provider-cloudflare/issues/725)) 
 * `resource/cloudflare_custom_ssl`: Allow `geo_restrictions` to be `nil` and not included in the request payload ([#714](https://github.com/cloudflare/terraform-provider-cloudflare/issues/714)) 
-* `datasource/cloudflare_zones`: Filtering is now performed on the server side and the `name` parameter is no longer a regex. Instead, `name` is a string to match on and `match` is a regex. See the website documentation for more examples and updated references ([#708](https://github.com/cloudflare/terraform-provider-cloudflare/issues/708)) 
+* `datasource/cloudflare_zones`: Filtering is now performed on the server side and the `name` parameter is no longer a regex. Instead, `name` is a string to match on and `match` is a regex. This may pose a breaking change in your codebases resulting in bogus CloudFlare API permission errors. See the website documentation for more examples and updated references ([#708](https://github.com/cloudflare/terraform-provider-cloudflare/issues/708)) in order to make your code compatible with this release.
```

Comment:
```suggestion
* `datasource/cloudflare_zones`: Filtering is now performed on the server side and the `name` parameter is no longer a regex. Instead, `name` is a string to match on and `match` is a regex. See the website documentation for more examples and updated references ([#708](https://github.com/cloudflare/terraform-provider-cloudflare/issues/708)) in order to make your code compatible with this release.
```

Reply:
That API permission error sent me on a wild goose chase that made me think it was an API issue rather than a compatibility issue with the recently release version of the provider - hence why I didn't check the CHANGELOG the first time. Others may go through the same.

If that's something you want out of the CHANGELOG entry, I would be ok with removing it, however I think it devalues the change.

Reply:
I understand what you're saying. The thing I want to avoid here is people coming across this changelog and start attributing permission errors to this single change. Unfortunately, edge cases causing permission errors are slightly more common since the launch of API tokens due to the number of configurations and options established around API keys and I wouldn't want to muddy the waters with a statement like the proposed one here.

Reply:
Ok. commited the proposed change. Hopefully SEO brings people to this PR if they face the same issue :smile: